packages/edit-post/src/components/visual-editor/index.js

```diff
@@ -28,6 +29,7 @@ function VisualEditor() {
 		<BlockSelectionClearer className="edit-post-visual-editor editor-styles-wrapper">
 			<VisualEditorGlobalKeyboardShortcuts />
 			<MultiSelectScrollIntoView />
+			<Popover.Slot name="block-toolbar" />
```

Comment:
`.edit-post-visual-editor` provides the `relative` positioned container within the scroll container that the popover needs.